src/test/java/uk/gov/hmcts/reform/translate/helper/DictionaryMapperTest.java

```diff
@@ -20,9 +21,11 @@ class DictionaryMapperTest {
     void shouldMapModelToEntityWithTranslationUploadEntity() {
         val dictionaryMapper = new DictionaryMapper();
         val currentPhrase = getCurrentPhrase();
+        val translationUploadEntity = new TranslationUploadEntity();
+        translationUploadEntity.setUserId("3333");
         val dictionaryEntity = dictionaryMapper.modelToEntityWithTranslationUploadEntity(
```

Comment:
This will not compile :(     dictionaryMapper.dictionaryMapper("3333") or dictionaryMapper.setUserId("3333") cannot be passed to   dictionaryMapper.modelToEntityWithTranslationUploadEntity(
            currentPhrase,
            dictionaryMapper.dictionaryMapper("3333"),  <---- 
            translationUploadEntity
        );